backend/src/test/java/com/timlenny/backend/service/TopicServiceTest.java

```diff
@@ -42,8 +42,6 @@ void isGetAllTopicsReturningAllTopics_whenGetAllTopicsCalledAndRepoIsEmpty() {
     void isAddTopicAddingNewTopic_whenAddTopicCalled() {
         when(topicRepository.save(demoTopicStart)).thenReturn(demoTopicStart);
         Topic actual = topicService.addTopic(demoTopicStartDTO);
-        assertEquals(actual, demoTopicStart);
+        assertEquals(actual.getTopicName(), demoTopicStart.getTopicName());
```

Comment:
You could use the topicName itself instead of Topic as actual. This makes the assertEquals cleaner.